packages/lesswrong/lib/sections.js

```diff
@@ -36,4 +46,8 @@ const postViewSections = {
   'all_drafts': 'all drafts',
 }
 
+if (getSetting('EAForum')) {
```

Comment:
This seems a little backwards (as in, you could add the settings here, rather than taking them away), but I get that it's nice to have all the declarations written together at the top. Not a defeater, but something to think about

Reply:
The picture is clearer if you look at our diff to LW before and after this PR. Now we don't have anything different within the frequently-changed `postViewSections`. We just remove the part we don't want based on a setting.